iiwa_driver/src/iiwa.cpp

```diff
@@ -304,8 +305,16 @@ namespace iiwa_ros {
         _fri_message_data->resetCommandMessage();
 
         if (_robot_state.getClientCommandMode() == kuka::fri::TORQUE) {
+            _joint_position_command = _joint_position;
```

Comment:
Last comment :stuck_out_tongue: : Here there is a full memory copy and possibly a new memory allocation? Not sure what `std` is doing on a copy vector constructor. Do we need this? Can we edit the for loop later on to be `_joint_position_command.at(i) = _joint_position.at(i) + ...`? Not sure if this is really going to make a difference (it is just 7 values after all).

Reply:
Fair point though. It's an unnecessary copy